students/alex_whitty/lesson4/trigrams.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+with open('double_take.txt', 'r') as f:
+
+    for line in f:
+        print(line, end='')
+
```

Comment:
You should turn file operations into a function that returns a list of words. This code would do the trick:

```
def parse_file():
    '''
    Takes a hardcoded file name and returns a list of words
    '''
    # Setup a translator to remove punctuation
    translator = str.maketrans('', '', string.punctuation)
    # Setup an empty list to contain the full list of words in the text
    full_list = []
    with open('double_take.txt', 'r') as f:
        for line in f:
            # This will take each line, remove punctuation and then split it
            # into a list of words and attach this list to the full_list
            # note the use of "extend()" instead of "append()"
            # TODO: Figure out how to remove carriage returns
            full_list.extend(line.translate(translator).split(' '))
    return full_list
```